trojsten/special/plugin_prask_7_2_1/static/plugin_prask_7_2_1/main.css

```diff
@@ -0,0 +1,45 @@
+html {
+    font-family: "Fira Code", monospace;
```

Comment:
Pouzivanie nahodnych fontov zhorsuje latency. Ked to tam velmi chces, tak vpohode, ja len aby si vedel.